LoopBackTests/bin/run-unittests

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -ex
+trap 'kill $(jobs -p)' EXIT
+
+# record the build environment
+xcodebuild -version
+xcodebuild -showsdks
+instruments -s devices
+
+# build the target
+xcodebuild \
+    -project LoopBack.xcodeproj \
```

Comment:
The only difference I can see between this `xcodebuild` invocation and the one that CI currently does is CI adds `-derivedDataPath $WORKSPACE/derived`. I'm not overly familiar with the build process, but one thing we could do to unify this script with our traditional build process would be to add an `$XCODEBUILD_ARGS` in here so that the extra parameter could be injected.